packages/docs/src/pages/guides/layers/guidelines.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Layers
+navigation:
+  - text: 'Code'
+    to: '/guides/layers/code'
+  - text: 'Guidelines'
+    to: '/guides/layers/guidelines'
+---
+
+## Using shadows to create layers
+
+Elements with shadows can help users understand an interface. Shadows indicate hierarchy of elements by differentiating between two objects. In some cases, shadows help users to understand that one object is above another.
+
+## Guidelines
+- Use q **z-depth 1** as a hover effect for cards.
+- Use q **z-depth 2** for megamenus or other dropdown menus.
+- Allow surfaces to move in front of and behind other surfaces, such as scrolling the page behind a dropdown menu.
```

Comment:
I'm curious about this --- the patterns I've seen with modals is to _prevent_ the background from scrolling behind the top of the stack.

Reply:
I wondered that too. I interpreted this as say you open the UniversalHeader dropdown menu then you should still be able to scroll on the page.

And agree that modals should prevent background scrolling.

Reply:
Removed that confusing bullet for now.